content/docs/extensions/model/index.md

```diff
@@ -0,0 +1,10 @@
+# Model extensions
+
+Model extensions add support for new types models that MLEM can covert into MLEM
+model objects in [`save` API method](/doc/api-reference/save)
+
+Typicaly they will implement [ModelType](/doc/user-guide/mlem-abcs#modeltype)
+and [ModelIO](/doc/user-guide/mlem-abcs#modelio) interfaces.
+
+Some also implement [DataType](/doc/user-guide/mlem-abcs#datatype) interface if
+specific data objects are needed for model to work.
```

Comment:
For now just a copy edit summarizing this part but let's consider (or make an issue about) auto-linking many of these ABCs as well as any other reference-like terms (commands, extensions even, other types, etc).

```suggestion
Typically they will implement [ModelType] and [ModelIO] interfaces. Some also
implement the [DataType] interface if specific MLEM data objects are needed for
model to work.

[modeltype]: /doc/user-guide/mlem-abcs#modeltype
[modelio]: /doc/user-guide/mlem-abcs#modelio
[datatype]: /doc/user-guide/mlem-abcs#datatype
```